pkg/storage/s3.go

```diff
@@ -378,28 +380,20 @@ func (rs *S3Storage) WalkDir(ctx context.Context, opt *WalkOption, fn func(strin
 	if opt == nil {
 		opt = &WalkOption{}
 	}
-
-	// NOTE: leave prefix empty if subDir is not set. Else, s3 will return empty result!
-	prefix := ""
-	if len(opt.SubDir) > 0 {
-		prefix = rs.options.Prefix + opt.SubDir
-		if len(prefix) > 0 && !strings.HasSuffix(prefix, "/") {
-			prefix += "/"
-		}
+	prefix := rs.options.Prefix + opt.SubDir
```

Comment:
What about `path.Join`?

Reply:
path.Join cannot work on windows since it separator is not "/".  And `options.Prefix` already contains the suffix "/"